src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/SerializationRulesDiagnosticAnalyzer.cs

```diff
@@ -25,16 +24,36 @@ public sealed class SerializationRulesDiagnosticAnalyzer : DiagnosticAnalyzer
                 nameof(MicrosoftNetCoreAnalyzersResources.ImplementSerializationConstructorsDescription),
                 MicrosoftNetCoreAnalyzersResources.ResourceManager, typeof(MicrosoftNetCoreAnalyzersResources));
 
-        internal static DiagnosticDescriptor RuleCA2229 = new DiagnosticDescriptor(RuleCA2229Id,
+        internal static DiagnosticDescriptor RuleCA2229Default = new DiagnosticDescriptor(RuleCA2229Id,
```

Comment:
Please switch to using `DiagnosticDescriptorHelper.Create` in NetAnalyzers project. You may want to pull latest sources from master and compare with other uses. I am planning to write an analyzer to Roslyn.Diagnostics.Analyzer to enforce this for our repo soon.